core/src/main/java/io/questdb/cairo/RebuildColumnBase.java

```diff
@@ -268,6 +271,10 @@ private void reindexOneOrAllColumns(
                     );
                 }
             }
+            //To validate if the table has any index or not
```

Comment:
nit: this comment is redundant as the code is self-explaining.

Reply:
The comment has been removed.